docs/sxpswap/users/trust-wallet.md

```diff
@@ -4,6 +4,34 @@ title: Swap/Migration for Trust Wallet users
 
 # Introduction
 
-!!! danger annotate "Coming soon"
+!!! danger annotate "Disclaimer!"
 
-    The implementation for using the swap bridge with the Trust Wallet is currently being implemented and will be available in early Q2.
+    Please read the information on this page carefully.
+    You need a Solar Wallet address to perform the swap from ERC20/BEP20 SXP to Mainnet SXP.
+    Visit our official website to download the latest [Solar Desktop Wallet](https://solar.org/desktop-wallet).
+
+
+## Swap procedure
+
+This guide is meant for users that have their ERC20/BEP20 SXP tokens in Trust Wallet.<br /> <br />
+Read the instructions below to swap your ERC20/BEP20 SXP tokens to Mainnet SXP coins:
+
+1. Create a mainnet address using the [Solar Desktop Wallet](https://solar.org/desktop-wallet) or [click here to learn more about creating wallets](/desktop-wallet/user-guides/create-import-wallets/).
+2. Make sure to securely store your passphrase with additional backups. Do not share this passphrase with anyone. 
+3. Make sure to have the SXP tokens (BEP20 or ERC20) in your Trust Wallet with gas to cover the fees for the swap.
+4. In the Trust Wallet app, click Browser. Then visit the Swap Website and make sure that you are visiting the right domain. [https://swap.solar.org](https://swap.solar.org)
```

Comment:
Links like these should open in a new tab because the destination URL is external.

`[https://swap.solar.org](https://swap.solar.org)` => `<a href="https://swap.solar.org" target="_blank" rel="noopener noreferrer">swap.solar.org</a>`